Project.toml

```diff
@@ -6,6 +6,7 @@ version = "0.6.1"
 [deps]
 CSV = "336ed68f-0bac-5ca0-87d4-7b16caf5d00b"
 CairoMakie = "13f3f980-e62b-5c42-98c6-ff1f3baf88f0"
+DataFrames = "a93c6f00-e57d-5684-b7b6-d8193f3e46c0"
```

Comment:
Is this really necessary? I think it's just use in tandem with CSV.jl, and we saw that those are quite easy to read, so I'd drop this dependency unless it's really necessary.